test/mongodb.ts

```diff
@@ -125,3 +196,8 @@ export * from '../src/write_concern';
 
 // Must be last for precedence
 export * from '../src/index';
+
+if (process.env.DISABLE_MONGODB_LEGACY !== 'true') {
```

Comment:
from slack, rename this variable to its logical opposite and default it to true with a TODO to remove the default in the ticket to convert callback tests

Reply:
done